src/preprocessor.py

```diff
@@ -0,0 +1,80 @@
+import numpy as np
+import cv2 as cv
+import os
+
+
+"""Requires: opencv-python, numpy"""
+class Preprocessor:
+    def __init__(
+        self,
+        contrast: float, invert_colors: bool,
+        denoise_intensity: int,
+        sharpen_intensity: float,
+        input_path: str, output_path: str = None
+    ):
```

Comment:
Can you add some default values?

Reply:
Sure, pushing soon

Reply:
Added [default parameters](https://github.com/AIVMZB/Latina/commit/7ea01c57763b1fdb168e4feae90bc198e61b13cc), as you asked